### ISSUE TITLE
False-positive for backslashes in multiline strings

### ISSUE THREAD

**Issue by orsinium** [sobolevn/flake8-broken-line#38]
Minimal code to reproduce:

```python
"""
\
"""
```
